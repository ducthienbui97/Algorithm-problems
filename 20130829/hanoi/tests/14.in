3
14
AB BC BA AC CA CB
14
BC CB CA AB BA AC
14
CA BA BC AC AB CB
