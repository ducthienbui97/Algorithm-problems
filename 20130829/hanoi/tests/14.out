16383
1594323
3188645
